utils/clean/clean.go

```diff
@@ -27,25 +27,25 @@ import (
 	"github.com/percona/pmm-managed/models"
 )
 
-// CleanResults has unexported fields for the results cleanup function.
-type CleanResults struct {
+// Service has unexported fields for the results cleanup function.
+type Service struct {
```

Comment:
But it is not what we call Service in pmm-managed, that's why it is in utils/clean package.
Let's revert this change to make PR smaller

Reply:
Reverted.